bears/general/KeywordBear.py

```diff
@@ -92,20 +97,21 @@ def run(self,
         '''
         comments = _get_comments(dependency_results)
 
-        simple_keywords_regex = re.compile(
-            '(' + '|'.join(re.escape(key) for key in keywords) + ')',
-            re.IGNORECASE)
-
-        message = "The line contains the keyword '{}'."
-        yield from self.check_keywords(filename, file, comments,
-                                       simple_keywords_regex, message)
-
-        if regex_keyword is not '':
-            regex = re.compile(regex_keyword)
-            message = ("The line contains the keyword '{}' which "
-                       'resulted in a match with given regex.')
-            yield from self.check_keywords(filename, file, comments, regex,
-                                           message)
+        if comments:
```

Comment:
well actually we still want the keywordbear to run even if it doesn't know where the comments are! Knowing the comments is optional and if they're not there the bear  should still work!